rs/ic_os/network/src/systemd.rs

```diff
@@ -120,7 +120,7 @@ pub fn generate_systemd_config_files(
     ipv6_address: &Ipv6Addr,
 ) -> Result<()> {
     let mut interfaces = get_interfaces()?;
-    interfaces.sort_by(|a, b| a.speed_mbps.cmp(&b.speed_mbps));
+    interfaces.sort_by(|a, b| b.speed_mbps.cmp(&a.speed_mbps));
```

Comment:
This now sorts the interfaces in _descending_ order so the fastest interface is first. Though I'm not sure this really matters because there should only be a single interface that's up.

Reply:
Done!